certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -3835,49 +3950,274 @@
           "200": {
-            "description": "Success"
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddAttachmentsToPdf"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddAttachmentsToPdf"
+            }
           },
           "400": {
-            "description": "Bad request"
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseAddAttachmentsToPdf"
+            }
           }
         },
-        "x-ms-visibility": "internal",
-        "summary": "Get Convert Cad Schema",
-        "description": "Provides the dynamic schema for the Convert CAD action."
+        "x-ms-visibility": "important",
+        "summary": "Add Attachments to PDF",
+        "description": "Add one or more attachments to the PDF file provided - https://support.encodian.com/hc/en-gb/articles/8087678202653"
       }
     },
-    "/api/v1/DynamicSchema/GetDynamicSchemaSignPdf": {
-      "get": {
+    "/api/v1/Core/ConvertJsonToExcel": {
+      "post": {
         "tags": [
-          "DynamicSchema"
+          "Core"
+        ],
+        "operationId": "ConvertJsonToExcel",
+        "consumes": [
+          "application/json"
         ],
-        "operationId": "GetDynamicSchemaSignPdf",
         "produces": [
           "application/json"
         ],
         "parameters": [
           {
-            "in": "query",
-            "name": "certificateType",
-            "type": "string",
-            "x-ms-summary": "Certificate Type",
-            "description": "The output format to obtain the dynamic schema for."
+            "in": "body",
+            "name": "operation",
+            "description": "Request object, refer to schema definition: DtoConvertJsonToExcel",
+            "schema": {
+              "$ref": "#/definitions/DtoConvertJsonToExcel"
+            }
           }
         ],
         "responses": {
           "200": {
-            "description": "Success"
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseConvertJsonToExcel"
+            }
           },
-          "400": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseConvertJsonToExcel"
+            }
+          },
+          "400": {
+            "description": "Bad request",
+            "schema": {
+              "$ref": "#/definitions/DtoResponseConvertJsonToExcel"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Convert JSON to Excel",
+        "description": "Convert JSON into a Microsft Excel file - https://support.encodian.com/hc/en-gb/articles/7690520790045"
+      }
+    },
+    "/api/v1/DynamicSchema/GetDynamicSchemaCreateBarcode": {
```

Comment:
@Encodian can you please confirm this change?
![image](https://user-images.githubusercontent.com/97551789/214429371-afdd5abd-6878-4734-8b7d-a15f1df21adc.png)

Reply:
Hi @Amjed-Ayoub - Apologies, this is resolved